### DISCUSSION TITLE
Plug-in automatic listing of backlinks

### DISCUSSION THREAD

**Discussion by Paunchboi** [zim-desktop-wiki/zim-desktop-wiki#2297] [Ideas]
Hi,

I am new to this so forgive me if I am posting this in the wrong place. Perhaps there is a way to do this or at least some work-around. However this is my issue that I have been searching in vain to find:

- I often put labels in my articles. And by labels I mean links to other articles.
- So for example lets say I write x articles named after animals: horses, goats, toads, lamas...
- I then create an article named "mammals"
- Now I write "mammals", Link it and copy paste it into articles "horses", "goats", "lamas"...
- When I go back to the article mammals I can see the backlinks right but if I'd want to create a list in the article I have to write entry by entry manually. It would be SO NICE to be able to just plop the whole list there, linked or not linked.

So that's my issue in full.
Is there a way to do this? If not, seems general enough to be a made into a plug-in.
Is there any work around that simplifies this task to any extent at all?

Thanks in advance!

**Comment by jaap-karssenberg** [comment]
Implementation would be rather straight forward. The plugin would need to
define aan online object with a widget that displays the list from the side
panel.

This way you have dynamic view that is always up to date.

On export it would be converted to a static list.

An alternative would be to allow to copy paste from the side panel to a
bullet list. But than it would not update dynamically.

Regards

Jaap

Op zo 15 jan. 2023 13:12 schreef Paunchboi ***@***.***>:

**Reply by introt** [reply]
For some use cases it would be nice to also have a way to copy search results as a list of filenames or links. This and that could then share the API, allowing those dynamic views to not only show backlinks, but also pages with a certain (set) of tags etc.

**Comment by Paunchboi** [comment]
Thanks for the replies guys! As far as I am concerned, even a static list would be tremendous help and time saving - a dynamic list even more so.
Judging from the tone it was a bit unclear for me what the likelihood of this being done and what that depends on in turn. I have no experience of programming although I certainly would like to learn. If it is up to me to get this done further guidance would be greatly appriciated :)
